src/e2e/scala/temple/SimpleE2ETestData.scala

```diff
@@ -245,4 +245,15 @@ object SimpleE2ETestData {
       |  isDefault: true
       |  editable: true
       |""".stripMargin
+
+  val prometheusConfig: String =
+    """global:
```

Comment:
This continues to baffle me, in my IntelliJ the `|` is aligned with the third `"`